app/src/main/java/com/mashup/mobalmobal/ui/profile/data/dto/ProfileResultDto.kt

```diff
@@ -0,0 +1,84 @@
+package com.mashup.mobalmobal.ui.profile.data.dto
+
+import com.google.gson.annotations.SerializedName
+import com.mashup.mobalmobal.dto.DonationDto
+import com.mashup.mobalmobal.ui.profile.domain.model.ProfileItem
+
+data class ProfileResultDto(
+    @SerializedName("result")
+    val result: Int,
+    @SerializedName("data")
+    val `data`: ProfileDetailDto
+)
+
+data class ProfileDetailDto(
+    @SerializedName("name")
+    val name: String,
+    @SerializedName("nick_name")
+    val nickName: String,
+    @SerializedName("profile_url")
+    val profileUrl: String,
+    @SerializedName("point")
+    val point: Double,
+    @SerializedName("my_donation_summary")
+    val donationSummary: DonationSummaryDto,
+    @SerializedName("my_request_donation")
+    val requestDonations: List<DonationDto>,
+    @SerializedName("my_donatings")
+    val donations: List<DonationDto>,
+    @SerializedName("message")
+    val message: String
+)
+
+data class DonationSummaryDto(
+    @SerializedName("request_count")
+    val requestCount: Int,
+    @SerializedName("donated_count")
+    val donatedCount: Int,
+    @SerializedName("closed_count")
+    val closedCount: Int
+)
+
+fun ProfileDetailDto.toProfileItems(): List<ProfileItem> =
+    listOfNotNull(
+        ProfileItem.User(
+            name,
+            nickName,
+            profileUrl
+        ),
```

Comment:
```suggestion
        ProfileItem.User(
            name = name,
            nickName = nickName,
            profileUrl = profileUrl
        ),
```
지금은 parameter들이 직관적이라 크게 상관없는데 kotlin 코드이고, 이처럼 parameter 하나 당 엔터 하나씩 들어가는 경우에는 parameter name을 명시해주는건 어떤가요?

Reply:
현재 파라미터랑 변수명이 같아서 이런식으로 했는데 name 명시해주도록 할게요~

Reply:
수정했어요~ 
커밋넘버 : c1da558